docs/reference/inference/put-inference.asciidoc

```diff
@@ -306,16 +306,6 @@ maximum token length. Defaults to `END`. Valid values are:
 For `openai` and `azureopenai` service only. Specifies the user issuing the
 request, which can be used for abuse detection.
 
-=====
```

Comment:
@timgrein This set of `=====` is the closing pattern of the section starting in line 282. The CI complains because it's missing. If you could put it back, the CI will pass.